addons/stock_dropshipping/tests/test_dropship.py

```diff
@@ -298,6 +298,49 @@ def test_correct_vendor_dropship(self):
         self.assertTrue((purchase.date_planned - purchase.date_order).days == 10, "The first supplier has a delay of 10 days")
         self.assertTrue(purchase.amount_untaxed == 8, "The price should be 4 * 2")
 
+    def test_kit_dropshipped_change_qty_SO(self):
```

Comment:
I would put the test in `mrp_subcontracting_dropshipping` to be sure to launch it